### ISSUE TITLE
[RoBERTa] the loss is increasing all of sudden at epoch 15

### ISSUE THREAD

**Issue by datquocnguyen** [pytorch/fairseq#1533]
Hi,
I am pre-training a RoBERTa language model. The training process of RoBERTa seems fine within 14 epochs (the loss keeps decreasing). However, at epoch 15, I find the loss is increasing significantly, valid loss increased from 1.8 at epoch 14 to 10.1 at epoch 15. Please see the attachement file for more details. 
I am wondering why it happended ? Is there any thing wrong with my training process?
Thanks a lot. 
[log_roberta_epoch15.txt](https://github.com/pytorch/fairseq/files/3991060/log_roberta_epoch15.txt)

**Comment by myleott** [comment]
The loss is exploding -- you probably need to decrease the learning rate or increase the batch size.

**Comment by datquocnguyen** [comment]
Thanks. I understand that the loss is exploding, the problem is why the loss **all of sudden** is exploding (it should not be like that). The training process runs totally fine in the first 14 epochs!

**Comment by myleott** [comment]
While it doesn't usually happen so late in training, we have seen this kind of thing before, especially when training with large learning rates (as we do with RoBERTa). If you still have the earlier checkpoints, you can try resuming training from one of them (when the loss was stable) but using a different `--seed`. That will change the data order and may avoid the explosion.

**Comment by datquocnguyen** [comment]
Thanks for your quick response. I will resume my training from epoch 14 (just a note that I already used a small batch size of 512 and small learning rate of 5e-5 due to a limited computation resource).
